src/pages/Orders.tsx

```
@@ -0,0 +1,843 @@
+  const { showNotification } = useNotification();
+
+  // Sort orders based on selected criteria
+  const sortedOrders = [...orders].sort((a, b) => {
+    switch (sortBy) {
+      case 'latest':
+        return new Date(b.created_at).getTime() - new Date(a.created_at).getTime();
+      case 'oldest':
+        return new Date(a.created_at).getTime() - new Date(b.created_at).getTime();
+      case 'status':
+        // Sort by status: pending -> processing -> shipped -> delivered -> cancelled
+        const statusOrder = { 'pending': 1, 'processing': 2, 'shipped': 3, 'delivered': 4, 'cancelled': 5 };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)